.github/workflows/ci.yml

```diff
@@ -0,0 +1,378 @@
+name: CI
```

Comment:
I find the name "CI" not explicit enough 🤔 

Maybe `Build and Test` or something like that?

Reply:
My concern was not to make the root title verbose because is is combined with the title of each sub-job in PR UI. 
<img width="845" alt="Screenshot 2024-03-27 at 11 04 13 AM" src="https://github.com/elastic/cloudbeat/assets/1380039/50fce26d-c54a-4973-bceb-2faffb3a32f3">


But sure why not.